packages/fluent-editor/src/modules/ai/constants.ts

```
@@ -0,0 +1,44 @@
+
+import {
+    RICH_CONTENT_ICON,
+    STREAMLINE_CONTENT_ICON,
+    SYMBOL_ICON,
+    TRANSLATE_ICON,
+  } from './icons'
+
+export const INPUT_PLACEHOLDER = '请输入问题或"/"获取提示词';
```

ESLint

[error] Extra semicolon.

(style/semi)